components/hackerDashboard/AdminDashboard.tsx

```diff
@@ -13,14 +13,15 @@ import ActionsWidget from "./widgets/admin/actions";
 type Props = {
   profile: Profile;
   events: Array<any>;
+  hackathonConstants: any;
```

Comment:
can we possibly type hackathon constants and events?

Reply:
yer

Reply:
gotcha